### ISSUE TITLE
[HELP NEEDED] Kartminer7 testers with Luma3DS!

### ISSUE THREAD

**Issue by zoogie** [zoogie/Kartminer7#16]
I'm looking for Luma3DS users who can test Kartminer7 and receive crash dumps that say anything BUT "process name: menu".

So please, upload your crash dumps if you can, ( sdmc:/luma/dumps/arm11/crash_dump_000000XX.dmp ). You may have to rename the extension to zip to upload them here.

Thanks for everybody's continued help! We'll get stable km7 hax yet!

UPDATE 8/21/23: Given the near zero interest in this thread, Kartminer7 progress is effectively halted. I absolutely need other people's input on these issues because I can't recreate and debug them on my own -- all 6 of my tester systems (US,EU,JP x old/new) run this exploit flawlessly and thus provide no useful info to me anymore.

**Comment by Brainboggler** [comment]
Trying to get cfw on original 3ds and having this issue. Would like to keep my MK7 save data, but I suppose I can wipe it. Any way I can help without wiping as of right now?
Physical copy, crashing on Street pass menu. Changing language as in the 3ds.hacks guide causes the Street pass fake Mii's to be selectable and causes you to go through the character, map, and kart selection.

**Comment by zoogie** [comment]
> Trying to get cfw on original 3ds and having this issue. Would like to keep my MK7 save data, but I suppose I can wipe it. Any way I can help without wiping as of right now? Physical copy, crashing on Street pass menu. Changing language as in the 3ds.hacks guide causes the Street pass fake Mii's to be selectable and causes you to go through the character, map, and kart selection.

This thread is only for people who can delete their saves. I'm mainly looking for digital game owners since they can safely backup their save (although physical w/cfw could do the same). There is no pressure whatsoever for you to delete your save if you consider it valuable. There will be others showing up later to help out that have the digital game, so no need to nuke your save.

**Comment by Devine3DS** [comment]
I'm helping someone who has a Japanese 3DS and it's always crashing on Streetpass Mii list. We deleted the Savegame but still same crashing.Badly we can't share crash data https://github.com/zoogie/Kartminer7/assets/139223121/bb27ea5d-7254-4737-a9a2-b364b12c1f75

**Comment by zoogie** [comment]
> 

Thanks for that info, was afraid of that.
Can you do me another favor? Have this user check Data Management -> Streetpass Management -> mk7 icon  and tell me how many "Data Transmissions" there are.

Quick question, are you "Jan" in the nintendo homebrew discord?

**Comment by Devine3DS** [comment]
I will tell her thank you


Yes I'm Jan :)

**Comment by Devine3DS** [comment]
![image](https://github.com/zoogie/Kartminer7/assets/139223121/1c3fa6ba-0f53-46f2-8e18-68ef8cf4080b)

**Comment by zoogie** [comment]
> ![image](https://user-images.githubusercontent.com/139223121/257993479-1c3fa6ba-0f53-46f2-8e18-68ef8cf4080b.jpeg)

Ok, one more thing then I'm out of ideas again. Have them deactivate this streetpass at that screen you've shown. Reinstall the exploit with the computer, then go through the steps again on the 3ds, but after selecting Mario Kart Channel, decline to reactivate Streetpass when the prompt shows up, then proceed pressing A until the exploit activates at Start Race: Go.

**Comment by InsetFlame** [comment]
Hey @zoogie I've been using this exploit with my physical cartridge (I don't care about the save state) and it's crashing before I can even select player/friend in streetpass list. Maybe I should use a different method because this may not be a kartminer problem, it could be a game problem. The thing is, I can't find a problem like this ever reported anywhere on the internet. So I've decided to come here before I try a different method. Hope this helps and hope you have an answer.

**Comment by zoogie** [comment]
> Hey @zoogie I've been using this exploit with my physical cartridge (I don't care about the save state) and it's crashing before I can even select player/friend in streetpass list. Maybe I should use a different method because this may not be a kartminer problem, it could be a game problem. The thing is, I can't find a problem like this ever reported anywhere on the internet. So I've decided to come here before I try a different method. Hope this helps and hope you have an answer.

These early crashes I have no idea how to fix because I have no realistic means of debugging them. None of my 3ds's have this issue and I'm just grasping at straws trying to guess what may be the hidden variable that makes the exploit fail on these other user's machines.   

The best alternative to just guessing is having someone with cfw and mk7 who can reproduce the early crash and send me luma crash dumps. Wish me luck with that one since most of these folks have no personal incentive to volunteer their time helping me given they already have their machine hacked.

**Comment by DanteS76** [comment]
Hey @zoogie im having it crash at the streetpass list (japenese old 3ds xl) and I can help out with whatever youd need. I have it physically but I dont really value the save at all so I can help out if needed.

**Comment by DanteS76** [comment]
> I'm looking for kartminer7 users whose attempts ALWAYS lead to a crash at the Streetpass List screen showing up, and who have the digital mk7 (maybe physical too if you would be cool with sacrificing your savegame). With and without luma cfw is fine.
> 
> My goal is to test a theory I have that mk7's regular savegame (not the extdata) may be the root of these Streetpass crashes.
> 
> Instructions are simple: backup your regular save using the home menu backup feature, delete your regular save (hold 4 face buttons ABXY at the initial game menu), and try the whole kartminer7 process again. Let me know the results here. If you have cfw, please post crash dumps here (if the process name is menu, count that as a successful attempt and no post needed).

alright, i just deleted the save data and went to restart this process and now it crashes on the streetpass screen even without the exploit so i have no idea why that is. im gonna test for a little bit and report findings, sorry in advance if nothing ends up being useful.

**Comment by InsetFlame** [comment]
> 

Thanks, and good luck. Also if it helps, I have an American model old 3dsXL

**Comment by DanteS76** [comment]
I've now used every method you've brought up (deleting save data and restarting process, as well as deactivating streetpass, reinstalling the exploit, and then not turning streetpass back on) to no avail I'm honestly not sure what the problem is, but I'm going to try this on a New Nintendo 3ds (as this one isnt hacked) and see if I can reproduce this crash to get some crash logs

**Comment by zoogie** [comment]
> crashes on the streetpass screen even without the exploit so i have no idea why that is

That's because the delete save function should only delete the regular save (inside the cart), and extdata (sd card /w exploit) should be left alone by the deletion. It's still important to reinstall the extdata save (.bat scripts) after you delete the reg save, and then test.

Also looking forward to your new3ds test. Thanks so much for helping!

**Comment by DanteS76** [comment]
> That's because the delete save function should only delete the regular save (inside the cart), and extdata (sd card /w exploit) should be left alone by the deletion.
yeah i had wiped my sd card i wanted to start from scratch, i just started the entire process over (from enabling streetpass in the first place) and it continued crashing so I was a bit confused but I think I figured it out. Anyways Ill have the crash logs up tomorrow if I can replicate this and hopefully it'll give you some better insight, I'm also still gonna try and see if I can get it to work on the old 3ds and will give updates here if I get it to work.

**Comment by Devine3DS** [comment]
Maybe @DanteS76 can hack the New3DS first and hack the old version with Kartdlphax. And try the KartMiner7 after that on the now modded Japan normal console again and send the crash log to @zoogie

**Comment by DanteS76** [comment]
yeah i was thinking of doing that as well

**Comment by zoogie** [comment]
NOTICE: I've changed the topic. I just need general crash testing now, and no more regular savegame sacrifices.
The crashes I'm looking for only need "Process name: MARIOKAR" as a requirement, *not* "Process name: menu"

For anybody still working on the previous category, go ahead and complete it and report it if you can! Thanks!

**Comment by DanteS76** [comment]
> Maybe @DanteS76 can hack the New3DS first and hack the old version with Kartdlphax. And try the KartMiner7 after that on the now modded Japan normal console again and send the crash log to @zoogie

the source 3ds i was planning on using seems to have an issue with its SD card reader, so i'm going to try and install custom firmware on the old 3ds using the flashcart method and getting the crash logs that way

**Comment by Devine3DS** [comment]
Here is a Crashdump of a Japanese 3DS with Crash on MiiList

![image](https://github.com/zoogie/Kartminer7/assets/139223121/073690d5-92ac-4292-945e-c4ec500c36ff)

[Dmp.zip](https://github.com/zoogie/Kartminer7/files/12476381/Dmp.zip)

Hope this does help :)

**Comment by Devine3DS** [comment]
Did this help you with the crashing Problem?

**Comment by zoogie** [comment]
> Did this help you with the crashing Problem?

It will in time I'm sure, and thank you, but this week I'm trying to finish up a long overdue, different exploit and get it out. And dealing with a personal issue at the same time. I'll come back to this.
